examples/mnist/README.md

```diff
@@ -57,3 +57,41 @@ Run the example:
 python mnist_with_visdom.py
 ```
 
+## Distributed training
+
+#### Usage:
+
+Train MNIST by using three GPUs in two nodes.
+
+* Single node, multiple GPUs (ex. a node has two GPUs):
+  
+   Open a terminal and run the example on the GPU 0 (process rank 0):
+   ```
+   python mnist_dist.py --world_size 2 --rank 0 --gpu 0 --dist_method=file:///tmp/tmp.dat
+   ```
+   and in another terminal, run the example on the GPU 1 (process rank 1):
+   ```
+   python mnist_dist.py --world_size 2 --rank 1 --gpu 1 --dist_method=file:///tmp/tmp.dat
+   ```
+
+* Multiple nodes with multiple GPUs  (ex. two nodes have two GPUs, respectively):
+
+  * Run the example by GPU-ID 0 in the primary node:
+  ```
+  python mnist_dist.py --world_size 4 --rank 0 --gpu 0
```

Comment:
@snowyday I'm trying to check this case on 2 nodes with world_size = 3. Did not manage to work... Probably, something is missing in the description

Reply:
@vfdev-5 Many thanks! Your nodes might not access the shared file system. 

I'd like to give a similar description of [ImageNet training in PyTorch](
https://github.com/pytorch/examples/tree/master/imagenet) as the following:

Node 0:
```
python mnist_dist.py --world_size 2 --rank 0 --gpu 0 --dist_method 'tcp://IP_OF_NODE0:FREEPORT' --dist_backend 'nccl' 
```
Node 1:
```
python mnist_dist.py --world_size 2 --rank 1 --gpu 0 --dist_method 'tcp://IP_OF_NODE0:FREEPORT' --dist_backend 'nccl' 
```

Additonally, please check [torch.distributed.init_process_group](https://pytorch.org/docs/stable/distributed.html).

The option 'backend' of 'init_process_group' supports [TCP initialization](https://pytorch.org/docs/stable/distributed.html#tcp-initialization) and  [Shared file-system initialization](https://pytorch.org/docs/stable/distributed.html#shared-file-system-initialization).

Please try them in your tow nodes. After that, I rewrite the README.md.

Reply:
@snowyday I tried the `dist_method` using shared-file system accessible from both nodes. Maybe I experience some internal problems with the node communication. 

> Additionally, to avoid confusing, I'll rename args.dist_method => args.init_method.

I think we can keep the same notations as in the ImageNet example.

Reply:
@snowyday finally, I found the problem and managed to run the example. Let me modify the usage, then you can approve the change.

Reply:
@vfdev-5 Your usages are  easily explanation for users. Thank you for your modification!